tutorials/international/index.md

```diff
@@ -25,6 +25,56 @@ supports. Flutter provides widgets and classes that help with
 internationalization and the Flutter libraries themselves are
 internationalized.
 
+### Setting up an Internationalized app: the `flutter_localizations` package
+
+By default Flutter only provides US English localizations. To add
+support for other languages, applications must specify some additional
+`MaterialApp` properties and include a separate package called
```

Comment:
what if you don't use MaterialApp ?

Reply:
I added a paragraph about that below the example.

Reply:
I've also added qualifying paragraph in the opening section.